nntrainer/src/conv2d_layer.cpp

```diff
@@ -278,18 +362,20 @@ Tensor Conv2DLayer::conv2d(Tensor in, Tensor kernel, unsigned int const *stride,
       for (unsigned int i = 0; i < channel; ++i) {
         for (unsigned int ki = 0; ki < k_height; ++ki) {
           for (unsigned int kj = 0; kj < k_width; ++kj) {
-            sum +=
-              kernel.getValue(0, i, ki, kj) * in.getValue(0, i, j + ki, k + kj);
+            sum += kernel[i * k_height * k_width + ki * k_width + kj] *
+                   in[i * height * width + (j + ki) * width + (k + kj)];
           }
         }
       }
-      output.setValue(0, 0, I, J, sum);
+      if (bias != 0.0)
+        sum += bias;
```

Comment:
Maybe unnecessary if-statement cuz it's 0?

Reply:
fixed. thanks.